test/test_cli/configs/another_configs/config_for_testing.py

```diff
@@ -0,0 +1,3 @@
+from biggerquery.configuration import BiggerQueryConfig
+
+test_config = BiggerQueryConfig('example_test_configuration')
```

Comment:
can you add relative import here to test if it still works?

Reply:
i added this import in another file - /configs/config_for_testing.py (it's better place for relative import to add)